buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -48,23 +49,25 @@ class AirbytePythonPlugin implements Plugin<Project> {
             command = "install -r requirements.txt"
             inputs.file('requirements.txt')
             outputs.file('build/installedlocalreqs.txt')
-            outputs.cacheIf { true }
+
+            // HACK: makes all integrations depend on installing requirements for bases. long term we should resolve deps and install in order.
```

Comment:
> long term we should resolve deps and install in order.

what will that look like? should we just have an `airbyte-integration.gradle` that handles this stuff? again. doesn't have to happen now, just want to understand how you're thinking about it and if possible create an issue with those thoughts. 

if you add issue also add its number to the comment.

Reply:
I'm still figuring out precisely how to do this. I'll post on here later with the improvement ticket, but this HACK should be gone by EOW.